src/components/blocks/HeaderBlock/HeaderBlock.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import HeaderBlock from './HeaderBlock';
+
+export default {
+  title: 'Block/HeaderBlock',
```

Comment:
Let's rename Block to Blocks

Reply:
![image](https://user-images.githubusercontent.com/86728658/153407274-57c3c994-81bc-457a-8919-1ba84e33183b.png)